GameServerLib/Logic/Game.cs

```diff
@@ -255,6 +257,15 @@ private bool HandleDisconnect(Peer peer)
                 {
                     PacketNotifier.NotifyUnitAnnounceEvent(UnitAnnounces.SummonerDisconnected, peerinfo.Champion);
                 }
+                _logger.LogCoreInfo("somebody disconnected");
+                disconnectedPlayers++;
+                var number = _playerManager.GetPlayers().Count - disconnectedPlayers;
+                _logger.LogCoreInfo(number.ToString());
+                if (number == 0)
+                {
+                    _logger.LogCoreInfo("no players connected");
+                    Program.SetToExit();
```

Comment:
Should capitalize the log message.
Also should consider makeing the Exit be per instance rather than static function but that might be part of another issue here. :)
But most importantly this should be optional...